File: Team_Workspace/dalia_ahmed/task13.py

class tracker:
    def __init__(self):
        print("Tracker Born")
    def __del__(self):
        print("Tracker Died")

print("Start")
def test_scope():
     t1=tracker()
     print("Inside Scope")
test_scope()
print("end of program")
#t1 died after func unlike if there is loop it wiil not die

File: Team_Workspace/Fatma_Ahmed/cpp_tasks/task13/task13.cpp
// task13.cpp : This file contains the 'main' function. Program execution begins and ends there.
//

#include <iostream>
using namespace std;

class Tracker {
public:
    Tracker() {
        cout << "Tracker Born \n";

    }
    ~Tracker() {
        cout << "Tracker Died \n";

    }

};
int main()
{
    cout << "Start\n";
    {
        Tracker t1;
        cout << "Inside Scope\n";

    }
    cout << "End\n";

}

